src/HelperPluginManager.php

```diff
@@ -400,7 +404,17 @@ public function injectEventManager($first, $second)
      */
     public function validate($instance)
     {
-        if (!$instance instanceof $this->instanceOf) {
+        if ('callable' === $this->instanceOf) {
+            if (is_callable($instance)) {
+                return;
+            }
+
+            $instanceOf = Helper\HelperInterface::class;
+        } else {
+            $instanceOf = $this->instanceOf;
+        }
```

Comment:
We can make this simpler by removing the `$instanceOf` property entirely. The logic then becomes:

```php
public function validate($instance)
{
    if (is_callable($instance) || $instance instanceof Helper\HelperInterface) {
       return;
    }
    // throw exception
}
```

I can do that on merge.